cmd/kogito/command/shared/install_data_index.go

```diff
@@ -27,19 +27,20 @@ import (
 )
 
 const (
-	defaultDataIndexReplicas = 1
+	defaultDataIndexReplicas = int32(1)
 )
 
 // installDefaultDataIndex shortcut to install the default Data Index in the given namespace.
 // Useful for quick starts, since the Data Index can be updated later to fulfill users needs.
 func installDefaultDataIndex(cli *client.Client, namespace string) error {
 	log := context.GetDefaultLogger()
 
+	replicas := defaultDataIndexReplicas
```

Comment:
can't you use the constant directly instead assign its value to a new env?

Reply:
That's because [the Go Language Specification (Address operators) does not allow to take the address of a numeric constant](https://stackoverflow.com/a/30716481/10322355). So created a local variable and assigned the constant value to it.